src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,18 @@
-import React from 'react';
+import { FC } from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Probs = {
+  comments: Comment[] | [];
```

Comment:
you don't need `| []` for comments
With Comment[] type comments can be array with comments or empty array